nmigen_boards/orangecrab_r0_2.py

```diff
@@ -66,8 +66,7 @@ class OrangeCrabR0_2Platform(LatticeECP5Platform):
         Resource("adc", 0,
             Subsignal("ctrl",     Pins("G1 F1", dir="o")),
             Subsignal("mux",      Pins("F4 F3 F2 H1", dir="o")),
-            Subsignal("sense_hi", Pins("H3", dir="i")),
-            Subsignal("sense_lo", Pins("G3", dir="i")),
+            Subsignal("sense", Pins("H3", dir="i"), Attrs(IO_TYPE="LVCMOS33D"))
```

Comment:
That's not a diffpair!

Reply:
Should the IO_TYPE be something different? In order for the ADC to work, sense_hi and sense_low need to be connected with a comparator

Reply:
Should be `DiffPairs("H3", "G3", dir="i")` at the very least. I have no special insight about `IO_TYPE`, that's up to the board designer mostly.